gems/sorbet/lib/hidden-definition-finder.rb

```diff
@@ -191,8 +191,8 @@ def serialize_constants(source, rbi, klass, is_singleton, source_symbols, rbi_sy
     ret = []
 
     rbi.each do |rbi_entry|
-      # skip synthetic constants
-      next if rbi_entry["name"]["kind"] == "UNIQUE"
+      # skip duplicated constant fields
+      next if rbi_entry["name"]["kind"] == "UNIQUE" and rbi_entry["kind"] == "STATIC_FIELD"
```

Comment:
Should we not also skip MANGLE_RENAME?

Reply:
We don't serialize the unique name kind, just that it's unique; should I add that to the serialization and use that?